Sources/Views/MessageInputBar.swift

```diff
@@ -74,7 +74,10 @@ open class MessageInputBar: UIView {
             }
             blurView.isHidden = !isTranslucent
             let color: UIColor = backgroundView.backgroundColor ?? .white
-            backgroundView.backgroundColor = isTranslucent ? color.withAlphaComponent(0.75) : .white
+            backgroundView.backgroundColor = isTranslucent ? color.withAlphaComponent(0.75) : color.withAlphaComponent(1.0)
+            
+            let bgColor: UIColor = backgroundColor ?? .inputBarGray
+            backgroundColor = isTranslucent ? bgColor.withAlphaComponent(0.75) : bgColor.withAlphaComponent(1.0)
         }
```

Comment:
We can't have this. If you run this you will see it breaks the translucent functionality. We already have:
```
/// Also sets the backgroundView's backgroundColor to the newValue
    open override var backgroundColor: UIColor? {
        didSet {
            backgroundView.backgroundColor = backgroundColor
        }
    }
```